src/System.Collections.Concurrent/tests/ConcurrentDictionary/ConcurrentDictionaryTests.cs

```diff
@@ -57,6 +57,73 @@ public static void TestBasicScenarios()
             Task.WaitAll(tks);
         }
 
+        [Fact]
+        public static void TestAddNullValue_ConcurrentDictionaryOfString_null()
+        {
+            // using ConcurrentDictionary<TKey, TValue> class
+            ConcurrentDictionary<string, string> dict1 = new ConcurrentDictionary<string, string>();
+            dict1["key"] = null;
+        }
+
+        [Fact]
+        public static void TestAddNullValue_IDictionaryOfString_null()
+        {
+            // using IDictionary<TKey, TValue> interface
+            IDictionary<string, string> dict2 = new ConcurrentDictionary<string, string>();
+            dict2["key"] = null;
+            dict2.Add("key2", null);
+        }
+
+        [Fact]
+        public static void TestAddNullValue_IDictionary_ReferenceType_null()
```

Comment:
This is failing on the .NET Framework.  I believe that's expected, because the bug being fixed here is exactly what this test is validating.  As such, it should have an attribute on it like:
```C#
[SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Missing bug fix in https://github.com/dotnet/corefx/pull/28115")]
```
so that the test isn't executed on netfx.  Lack of this is why the netfx test run failed.